testing/features/support/core_ext/capybara/selenium/driver/firefox_driver.rb

```diff
@@ -8,7 +8,7 @@
 # for more details/discussion including possible other fixes.
 #
 # LH - Sep 2020
-module Capybara::Selenium::Driver::FirefoxDriver
+module Capybara::Selenium::Driver::FirefoxDriver # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
Wasn't sure if I should just change this syntax, but disabled the warning for now.

Reply:
So I'm not a fan of in-line stuff. Instead we can use the `--auto-gen-config` stuff. Or if we're comfortable we can add it to an AllCops exclude. The reason being these aren't picked up by the CLI. or easy to track down.

As a brief intro (Although it's written above). This is a copy+paste plus bugfix from an issue in Capybara. I believe it is now fixed in master and will be released in v3.33 (But we're capped at 3.29 currently).